pwa/app/lib/matchUtils.ts

```diff
@@ -16,3 +25,48 @@ export function sortMatchComparator(a: Match, b: Match) {
   }
   return a.set_number - b.set_number || a.match_number - b.match_number;
 }
+
+export function matchTitleShort(match: Match, event?: Event): string {
```

Comment:
Why is `event` optional? Can make it not optional (including in `MatchResultsTableProps`)?

Reply:
At the time I wasn't sure if we would always have the full event context when calling this. I suppose we can make it optional when we encounter that situation though.